hippod/mime_data_db.py

```diff
@@ -141,6 +141,32 @@ def decode_and_write_file(app, sha_sum, data, compress=False):
     return d
 
 
+def already_in_data_db(app, sha):
+    data_db = app['DB_DATA_PATH']
+    data_list = os.listdir(data_db)
+    if sha in data_list:
+        return True
+    return False
+
+
+def higher_mime_type_preference(app, sha, data):
+    # FIXME: hardcoded stuff
+    if data['mime-type'].startswith('x-snippet') or data['mime-type'].startswith('image'):
```

Comment:
Because for an image higher_mime_type_preference() would return False. In the if-condition in save_object_item() on line 191 it would jump to the else block, where the whole saving process with calculating size-real etc is skipped. In the container/achievement meta these keys wouldn't be listed. But I realized, thats the wrong logic...thats why the FIXME is there. I have to do the whole saving issue, too. Except not touching the attr.db file.